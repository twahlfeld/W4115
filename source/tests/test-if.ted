int main() {
int x;
x = 42;
    if (x == 42) {
        print(stdout, "ted\n")
    }
    return 0;
}
