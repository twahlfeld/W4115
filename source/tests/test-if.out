ted ted ted ted ted ted ted
